rs/nns/governance/src/lib.rs

```diff
@@ -246,6 +248,20 @@ pub fn temporarily_disable_active_neurons_in_stable_memory() -> Temporary {
     Temporary::new(&ACTIVE_NEURONS_IN_STABLE_MEMORY_ENABLED, false)
 }
 
+pub fn use_stable_memory_following_index() -> bool {
+    USE_STABLE_MEMORY_FOLLOWING_INDEX.with(|ok| ok.get())
+}
+
+/// Only integration tests should use this.
+pub fn temporarily_enable_stable_memory_following_index() -> Temporary {
+    Temporary::new(&USE_STABLE_MEMORY_FOLLOWING_INDEX, true)
+}
+
+/// Only integration tests should use this.
+pub fn temporarily_disable_stable_memory_following_index() -> Temporary {
+    Temporary::new(&USE_STABLE_MEMORY_FOLLOWING_INDEX, false)
+}
```

Comment:
is there another way to do this that guarantees the functions aren’t called in production?

Reply:
Testing out putting them behind compile-time flags.